backend/pyproject.toml

```
@@ -32,9 +32,9 @@ dependencies = [
     "python-socketio==5.9.0", # For log_events
     "social-auth-app-django==5.3.0", # For OAuth
     "social-auth-core==4.4.2", # For OAuth
-    "unstract-sdk~=0.57.0rc2",
-    "unstract-sdk[aws]~=0.57.0rc2",
-    "unstract-sdk[gcs]~=0.57.0rc2",
+    "unstract-sdk~=0.57.0rc3",
+    "unstract-sdk[aws]~=0.57.0rc3",
+    "unstract-sdk[gcs]~=0.57.0rc3",
```

```suggestion
    "unstract-sdk[gcs,aws]~=0.57.0rc3",
```
Please check and update the same in other files also similarly @pk-zipstack

---

Sure @chandrasekharan-zipstack

---

mahmoodmohaghegh65@gmail.com